packages/react-scripts/scripts/lint.js

```diff
@@ -0,0 +1,26 @@
+// @remove-on-eject-begin
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @remove-on-eject-end
+'use strict';
+
+const spawn = require('react-dev-utils/crossSpawn');
+const args = process.argv.slice(2);
+
+const result = spawn.sync(
+  'node',
+  [require.resolve('eslint/bin/eslint')]
+    .concat([
+      'src/**/*.{js,jsx,mjs}',
+      '--config',
+      require.resolve('../config/eslint.json'),
```

Comment:
i think this line can just be 
```
    require.resolve('eslint-config-react-app')
```

Reply:
after eject it needs to be 
```js
   require.resolve(paths.appPackageJson),
``` 
though

Reply:
i think we can do it by playing around with eject comments 😅 
```js
// @remove-on-eject-begin
/**
 * Copyright (c) 2015-present, Facebook, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
// @remove-on-eject-end
"use strict";

const spawn = require("react-dev-utils/crossSpawn");
const args = process.argv.slice(2);
// @remove-on-eject-begin
/**
// @remove-on-eject-end
const paths = require("../config/paths");
// @remove-on-eject-begin
*/
// @remove-on-eject-end
// @remove-on-eject-begin

const result = spawn.sync(
  "node",
  [require.resolve("eslint/bin/eslint")]
    .concat([
      "src/**/*.{js,jsx,mjs}",
      "--config",
      // @remove-on-eject-begin
      /**
      // @remove-on-eject-end
       require.resolve(paths.appPackageJson),
       // @remove-on-eject-begin
       */
      // @remove-on-eject-end
      // @remove-on-eject-begin
      require.resolve("eslint-config-react-app")
      // @remove-on-eject-end
    ])
    .concat(args),
  { stdio: "inherit" }
);

process.exit(result.status);
```

Reply:
or just 
```js
// @remove-on-eject-begin
/**
 * Copyright (c) 2015-present, Facebook, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
// @remove-on-eject-end
"use strict";

const spawn = require("react-dev-utils/crossSpawn");
const args = process.argv.slice(2);

const result = spawn.sync(
  "node",
  [require.resolve("eslint/bin/eslint")]
    .concat([
      "src/**/*.{js,jsx,mjs}",
      // @remove-on-eject-begin
      "--config",
      require.resolve("eslint-config-react-app")
      // @remove-on-eject-end
    ])
    .concat(args),
  { stdio: "inherit" }
);

process.exit(result.status);
```

Reply:
Thank you so much! Suggestion to use`require.resolve('eslint-config-react-app')` as a config path is a gold.